apps/explorer/src/pages/epochs/EpochDetail.tsx

```diff
@@ -91,14 +91,16 @@ export default function EpochDetail() {
     return (
         <div className="flex flex-col space-y-16">
             <div className="grid grid-flow-row gap-4 sm:gap-2 md:flex md:gap-6">
-                <EpochProgress
-                    epoch={epochData.epoch}
-                    inProgress={isCurrentEpoch}
-                    start={Number(epochData.epochStartTimestamp)}
-                    end={Number(
-                        epochData.endOfEpochInfo?.epochEndTimestamp ?? 0
-                    )}
-                />
+                <div className="flex max-w-[240px]">
```

Comment:
The epoch progress itself has a min-w, so maybe throw this max-w there too? Or hoist the min-w up here too